packages/libs/wdk-client/src/Views/Records/RecordTable/RecordFilter.tsx

```diff
@@ -24,7 +24,7 @@ type RecordFilterSelectorProps = {
 interface RecordFilterProps
   extends Omit<
     RecordFilterSelectorProps,
-    'toggleFilterFieldSelect' | 'containerClassName'
+    'toggleFilterFieldSelector' | 'containerClassName'
```

Comment:
This is a bug fix that didn't show up because the only consuming component was in a JSX file.